assets/scss/app.scss

```diff
@@ -11,6 +11,14 @@ $path: "../img/";
   @include phase-banner(beta);
 }
 
+.form-date {
+  input {
+    &[id$="-year"] {
+      width: 70px;
```

Comment:
Isn't px a fix size? should we be going for em or percentage for an accessibility/responsive design point of view? http://kyleschaeffer.com/development/css-font-size-em-vs-px-vs-pt-vs/